npm/design-system/src/components/virtualizedTree/VirtualizedTree.stories.tsx

```diff
@@ -66,7 +67,7 @@ const tree: TreeParent = {
   ],
 }
 
-export const VirtualizedTree = createStory(() => {
+export const VirtualizedTree: Story = createStory(() => {
```

Comment:
This cast should be unnecessary

Reply:
It actually is, because createStory returns `Story<unknown>` I am wondering what would type the Story here.

Reply:
I updated `utils.ts` a bit so that they become unnecessary.